File: py/src/base62/base62.py
CHARSET = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"


def is_valid_base62(input: str) -> bool:
    """
    Returns `True` if all characters in the input string are Base-62, `False` in all other cases
    """
    return bool(input) and all(c in CHARSET for c in input)

File: golang/base62.go
package golang

import (
	"slices"
)

var CHARSET = []rune("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")

// IsValidBase62 returns true if all characters in input are Base-62, false in all other cases
func IsValidBase62(input string) bool {
	if len(input) == 0 {
		return false
	}
	for _, c := range input {
		found := slices.Contains(CHARSET, c)
		if !found {
			return false
		}
	}
	return true
}
